### PR TITLE
View meeting locations

### PR DESCRIPTION
**Pull request by theresenieva**
Create page for when attendees are viewing an event's proposed locations. 

This page allows viewers to view locations, see number of votes vs. location and vote on locations. I have chosen to have the bargraph and the map displayed at the same time as this would work well with the real-time updating in the final product.

### CODE REVIEW

File: prototype/src/main/webapp/chooseLocation.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
   <body onload="createMap();">
     <h1>Meeting Locations: </h1>
     <p>Click on the map to propose a new location.</p>
-    <div id="map"></div>
+    <div id="map"></div></br>
+    <button id="myButton" class="float-left submit-button" >Send Event</button>
+    <script type="text/javascript">
+    document.getElementById("myButton").onclick = function () {
+        location.href = "/vote-locations.html";
```
Comment (vivianho18) [inline prototype/src/main/webapp/chooseLocation.html]: nit: 2 spaces for indentation

**Review by vivianho18 — COMMENTED** [review]
LGTM!

**Comment by anna-giang** [comment]
LGTM too! 😊

File: prototype/src/main/webapp/chooseLocation.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
   <body onload="createMap();">
     <h1>Meeting Locations: </h1>
     <p>Click on the map to propose a new location.</p>
-    <div id="map"></div>
+    <div id="map"></div></br>
+    <button id="myButton" class="float-left submit-button" >Send Event</button>
```
Comment (ghost) [inline prototype/src/main/webapp/chooseLocation.html]: What are these classes for?

File: prototype/src/main/webapp/chooseLocation.js [hunk 3]
```diff
@@ -28,7 +37,8 @@ function createMarkerForDisplay(lat, lng, content) {
     '<div id="bodyContent">' +
     "<p>Number of Votes: 1</br>" +
     "Voters: You</br></p>" +
-    "<button>VOTE</button><br/><br/>"
+    "<input type=\"checkbox\" id=\"vote\" name=\"vote\" value=\"vote\">" +
+    "<label for=\"vote\"> VOTE</label><br></br>" +
```
Comment (ghost) [inline prototype/src/main/webapp/chooseLocation.js]: I would use single quotes here (and also change the rest of the strings here to use single quotes) so you don't have to keep escaping the double quotes. Alternatively, use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), and you won't need the concatenations anymore.

**Review by ghost — APPROVED** [review]

File: prototype/src/main/webapp/chooseLocation.html (hunk 2 above)
Comment (theresenieva) [reply prototype/src/main/webapp/chooseLocation.html]: It was from an example online but I don't think it is necessary, I will remove it

File: prototype/src/main/webapp/chooseLocation.js (hunk 3 above)
Comment (theresenieva) [reply prototype/src/main/webapp/chooseLocation.js]: Thanks Austin - I'll update it :)

**Pull request merged**